lib/redis-manager.js

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable global-require */
+/* eslint-disable import/no-dynamic-require */
+
+'use strict';
+
+const { promisify } = require('util');
+const md5 = require('md5');
+const path = require('path');
+const logger = require('@janiscommerce/logger');
+const redis = require('redis');
```

Comment:
```suggestion
const redis = require(path.join(process.cwd(), 'node_modules', 'redis'));
```
Es necesario agregar las rutas a las dependencias dentro del process cwd ya que no estan como dependencias del paquete.